swan-lake/learn/testing-ballerina-code/testing-quick-start.md

```diff
@@ -21,52 +21,52 @@ To get started, let's write a simple Ballerina function and test it.
 1. First, let’s create a Ballerina project and add a new module. Use the `ballerina new` command to create the project. 
 For more information on the command, see [Structuring Ballerina Code](/swan-lake/learn/structuring-ballerina-code/).
 
-    The standard project will have the structure below.
+    Following is the structure of a standard project with the default module. In this example, default module has the 
+    source file ***main.bal*** and the test file ***main_test.bal***.
 
     ```bash
     project-name/
-        Ballerina.toml         
-        src/
-            module1/           	
-                main.bal   
-                Module.md
-                [resources/]
-                tests/
-                    main_test.bal
-                    [resources]	   
+        Ballerina.toml
+        Package.md
+        Module.md
```

Comment:
Let's remove the `[modules]` and `Module.md`

Reply:
@hevayo should we keep the `Package.md`. since only a lib template would have it? Also, anyway for this release, we don't generate the `Package.md` in new and init commands.